src/classes/UTIL_ACCT_Naming.cls

```diff
@@ -78,11 +78,14 @@ public class UTIL_ACCT_Naming {
         }
 
         for (Account acc: [SELECT Id, Name, RecordTypeId,
-                           (SELECT Id, LastName, Salutation, FirstName FROM Contacts WHERE AccountId IN :accountIdPrimaryContactId.keySet())
-                           FROM Account WHERE Id IN :accountIdPrimaryContactId.keySet()])
+                                  (SELECT Id, LastName, Salutation, FirstName, Account.RecordTypeId 
+                                  FROM Contacts 
+                                  WHERE AccountId IN :accountIdPrimaryContactId.keySet())
```

Comment:
I think this line is not needed. We shouldn't have to add this filter since this subquery only applies to the accounts you're querying. The will already be filtered by the account they belong to.